File: flight_most_traveled_destinations/v1/get/Python SDK/flight_most_traveled_destinations.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Where were people flying to from Madrid in the January 2017?
    '''
    response = waconvo.travel.analytics.air_traffic.traveled.get(originCityCode='MAD', period='2017-01')
    print(response.data)
except ResponseError as error:
    raise error

File: flight_most_traveled_destinations/v1/get/Node SDK/flight_most_traveled_destinations.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Where were people flying to from Madrid in the January 2017?
    const response = await waconvo.travel.analytics.airTraffic.traveled.get({
      originCityCode: "MAD",
      period: "2017-01",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
